node/src/options.rs

```diff
@@ -51,16 +48,13 @@ pub struct Options {
 pub enum Command {
     /// Create a configuration file.
     CreateConfig,
-    Run(RunOptions),
+    Mainnet(RunOptions),
+    Regtest(RegtestOptions),
```

Comment:
What about testnet? I think the model is incomplete. There can be nets other than mainnet that are not regtest.